nixos/modules/services/misc/evremap.nix

```
@@ -63,6 +63,18 @@ in
             '';
           };
 
+          phys = lib.mkOption {
+            type = lib.types.nullOr lib.types.str;
+            default = null;
+            example = "usb-0000:07:00.3-2.1.1/input0";
+            description = ''
+              If you have multiple devices with the same name, you can optionally
+              specify this attribute.
+
+              If available, the value will be printed when running `evremap list-devices` with elevated permissions.
```

"If available" sounds a bit odd.
```suggestion
              The physical device name to listen on.
              
              This attribute may be specified to disambiguate multiple devices with the same device name.
              The physical device names of each device can be obtained by running `evremap list-devices` with elevated permissions.
```